lib/Elastica/Query/AbstractGeoShape.php

```diff
@@ -4,33 +4,41 @@
 /**
  * geo_shape query.
  *
- * Query pre-indexed shape definitions
- *
  * @author Bennie Krijger <benniekrijger@gmail.com>
  *
  * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-shape-query.html
  */
 abstract class AbstractGeoShape extends AbstractQuery
 {
+    /** Return all documents whose geo_shape field intersects the query geometry. (default behavior) */
     const RELATION_INTERSECT = 'intersects';
+
+    /** Return all documents whose geo_shape field has nothing in common with the query geometry. */
     const RELATION_DISJOINT = 'disjoint';
-    const RELATION_CONTAINS = 'within';
```

Comment:
constant name didn't fit the value. Changing this should not break BC because it didn't work anyway so far as the relation was wrongly serialized.